web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -58,13 +59,22 @@ export const SidebarCardsList = (props: SidebarCardsListProps): ReactElement =>
     );
   };
 
+  const scrollAccordionToTop = (element: HTMLElement | null, { isDesktop }: { isDesktop: boolean }): void => {
```

Comment:
```suggestion
  const scrollAccordionToTop = (element: HTMLElement | null, isDesktop: boolean): void => {
```
[dust] My personal preference for this would be to make this just a boolean, not an object. It would still get passed to `scrollToTopOfElement` the same way, but in the context of this function I just think this makes more sense.

Reply:
Strong disagree. I think the team has discussed this before, but I really believe that passing raw booleans to any function ever is an anti-pattern, and they should always be parameterized/labelled like AnJu has here. 

If you change this, then when you call the function, it would look like: `scrollToTopOfElement(element, true)` and there is no way of knowing what the true/false boolean is controlling.